processor/attributesprocessor/factory.go

```diff
@@ -129,9 +130,10 @@ func buildAttributesConfiguration(config Config) ([]attributeAction, error) {
 			} else {
 				action.FromAttribute = a.FromAttribute
 			}
-
-		case DELETE:
-			// Do nothing since `key` is the only required field for `delete` action.
+		case HASH, DELETE:
+			if a.Value != nil || a.FromAttribute != "" {
+				return nil, fmt.Errorf("error creating \"attributes\" processor. Action \"HASH\" does not use \"value\" or \"from_attribute\" field. These must not be specified for %d-th action of processor %q", i, config.Name())
```

Comment:
nit: now it could be a DELETE too, "The selected action ... " will be fine if getting the exact one is cumbersome.

Reply:
Oh, missed that.